sql-join/big-spenders.sql

```diff
@@ -0,0 +1,4 @@
+  select "customers" as "customer",
+         "payments" as "payment"
```

Comment:
For this we just want the `firstName`, `lastName` and payment `amount`.

Reply:
Made the requested changes.